src/intents.js

```diff
@@ -165,6 +186,10 @@ export function createService (cozy, intentId, serviceWindow) {
               type: `intent-${intent._id}:error`,
               error: errorSerializer.serialize(error)
             }),
+            setSize: (doc) => setSize({
```

Comment:
`setSize()` signature should have a document as parameter, but something like `dimensions`.